### DISCUSSION TITLE
MoorDyn surge damping

### DISCUSSION THREAD

**Discussion by thanasisder** [OpenFAST/openfast#2050] [General]
Hello,
I am using OpenFAST with second-order hydrodynamics to study the motions of a SPAR FOWT. I am using external wave elevation (option 5) with experimental timeseries and comparing the results of the platform motions with experiments.

I also compared the results obtained by using MAP as a mooring model and MoorDyn and I see that the latter case introduces a signifcant damping of the surge motion. You can see the figure below. I attach the MoorDyn file as well, based on which I created the MAP file. The mooring at its current setup was tested and its equivalent linear stiffness matches the experimental arrangement up to 16 m of offset.

Could you please let me know what parameters could I change so that MoorDyn doesn't damp so much the surge motion? As you can see heave and pitch are improved very much when I use it instead of MAP, however for surge I have this problem.

Best,
Thanasis

![image](https://github.com/OpenFAST/openfast/assets/62059192/fe2e2c74-3f83-47de-86bf-77c6c3866a91)

**MoorDyn File** 
----------------------- LINE TYPES ------------------------------------------
1        NTypes    - number of LineTypes
Name     Diam      MassDen      EA    BA/-zeta  Can   Cat   Cdn   Cdt
(-)       (m)      (kg/m)       (N)   (N-s/-)   (-)   (-)   (-)   (-)
main     0.266      440        1.65E9   -2.0     0.765   0.383   1.244   0.383
---------------------- CONNECTION PROPERTIES --------------------------------
9       NConnects - number of connections including anchors and fairleads
Node    Type      X        Y        Z       M     V      FX   FY   FZ   CdA   CA
(-)     (-)      (m)      (m)      (m)     (kg)  (m^3)  (kN) (kN) (kN) (m^2)  (-)
1      Fixed   -320.6   -552.0    -200.0     0      0      0    0    0    0     0
2      Fixed   -320.6    552.0    -200.0     0      0      0    0    0    0     0
3      Fixed    639.2   0.0      -200.0     0      0      0    0    0    0     0
4      Connect  -19.65  -34.0     -46.6    3.15E+4  0      0    0    0    0     0
5      Connect  -19.65   34.0     -46.6    3.15E+4  0      0    0    0    0     0
6      Connect  39.3     0.0      -46.6    3.15E+4  0      0    0    0    0     0
7      Vessel   -10.44   0.0      -13.4      0      0      0    0    0    0     0
8      Vessel   5.22    -9.04     -13.4      0      0      0    0    0    0     0
9      Vessel   5.22     9.04     -13.4      0      0      0    0    0    0     0
---------------------- LINE PROPERTIES --------------------------------------
9        NLines    - number of line objects
Line    LineType  UnstrLen  NumSegs   NodeAnch  NodeFair  Flags/Outputs   calcChans
(-)       (-)       (m)       (-)       (-)       (-)         (-)            (-)
1         main     637.0      75      1         4           p                0
2         main     637.0      75      2         5           p                0
3         main     637.6      75      3         6           p                0
4         main     48         6       4         7           p                0
5         main     48         6       4         8           p                0
6         main     48         6       5         9           p                0
7         main     48         6       5         7           p                0
8         main     48         6       6         8           p                0
9         main     48         6       6         9           p                0

**Comment by RBergua** [comment]
I understand that MAP++ (quasi-static approach) does not account for the mooring line inertial forces and the hydrodynamics (e.g., drag and added mass). If you want MoorDyn to have less damping and behave closer to MAP++, you could reduce the hydrodynamic drag (Cdn parameter).

**Reply by jjonkman** [reply]
Dear @thanasisder,

 I agree with @RBergua.  You could also see if reducing hydrodynamic drag tangential to the line (`Cdt`) and structural damping of the lines (`zeta`) influences the results.

Best regards,

**Comment by thanasisder** [comment]
Dear @RBergua and @jjonkman,
Thank you very much both for your response!
In the same topic, I notice at some instants of my simulations where slow-drift motion should be significant that this is not captured by OpenFAST (figure below).

![image](https://github.com/OpenFAST/openfast/assets/62059192/3f1fbf62-1390-4b49-90ba-0c969e7ce43e)

While in heave and pitch the inclusion of the QTF improves the results a lot, it seems like that's not the case for surge. Supposed that the QTF is correct as indicated by the performance in the other two DOFs, my possible explanation is that there might be a parameter that is desactivated and ignores very low frequencies. For reference the system has a natural  period of 100 s for surge and around 30 seconds. Also WvLowCOff and WvHiCOff are set to 0 and 10 respectively.

So my questions here are: a) Does the inclusion of the difference frequency QTF shall be used always with activation of the second order wave elevation (I use external time history)  b) is the radiation kernel used for the calculation of the second order response in a way that I shall increase this time parameter ?

Thank you in advance!
Thanasis

**Reply by jjonkman** [reply]
Dear @thanasisder,

I have not done much work recently on second-order effects on spars, but from our experience on semisemubmersibles, the QTF may not fully capture all effects contributing to resonant excitation of the surge mode.  Nonlinear effects such as wave stretching and viscous effects may have a sizeable contribution.  The following paper exploring this for the OC4-DeepCwind semisubmersible may be of interest to you: https://www.sciencedirect.com/science/article/pii/S0960148122000635.

You could certainly check the sensitivity of your simulation results to the wave-radiation analysis time (`RdtnTMax`), but you shouldn't need to set `RdtnTMax` longer than it takes for the radiation impulse-response function to decay to near zero.

Best regards,

**Reply by thanasisder** [reply]
Dear @jjonkman,
Yes I am aware of this work on semisubmersibles, I just thought that since there was not much discussion about spars, the QTF is adequate in those cases. Thank you for your recommendations!
Best,
Thanasis
